fecgov/fec-cms
# [DO NOT MERGE] Pres Fin Map Data Cand List, Year Toggle

## Summary

- Resolves #3513 

Just a PR to verify that the list of candidates is correctly loading and displaying its data. If I tagged you and you don't care, just remove yourself.

## Impacted areas of the application
Rather than list all the disclaimers, it's easier to list what _is_ happening and what to test:
- the year toggle
- the list of candidates
bonus:
- clicking candidates will update the breadcrumb (kinda)
- adding other years to `availElectionYears` (and rebuilding) should change the year toggles...though we have no data for other years

Ignore everything else (style, location, order, every other component, 99% of code comments)

## Screenshots
![image](https://user-images.githubusercontent.com/26720877/74160700-cacc3600-4beb-11ea-8d04-8451927d1769.png)

## Related PRs
None yet

## How to test
- Pull, build, and run as normal
- Check http://127.0.0.1:8000/data/candidates/president/presidential-map/
- (You'll probably get a few warnings in the console about sameSite, jQuery CSP, non-unique #query, CORB)
- After a second or three (usually super fast), verify that
  - the candidate list data has loaded, that the information is as the API returns
  - the correct dollar figures have loaded and displayed
  - that the list is sorted by $, descending

(My apologies for asking you to ignore probably everything else, I know it's painful)
____
